cli/src/print.rs

```diff
@@ -1,5 +1,6 @@
 use {
     comfy_table::{modifiers::UTF8_ROUND_CORNERS, presets::UTF8_BORDERS_ONLY, Row, Table},
+    gluesql_core::ast::DataType,
```

Comment:
unit test module has independent scope, you need to move this import to 
![image](https://user-images.githubusercontent.com/2025065/163702467-1a0f3a07-5051-4cf6-9622-69483041617c.png)

Reply:
Thanks.. I knew it was something like what you suggested.  Nothing I tried seemed to work.   Now i"m having an issue with the test-suite-src/showcolumns.rs tests.  I've tried creating my own showcolumns macro, but what is occuring in the (copied from select macro) is a bit beyond my skills at this time.  Can you take a look?
Thanks!